src/main/minio.js

```diff
@@ -403,7 +403,6 @@ export class Client {
           this.logHTTP(reqOptions, null, e)
           cb(e)
         })
-      req.once('finish', () => pipe.removeAllListeners('error'));
```

Comment:
Now i remember it was related to a bug which was experienced where minio-js was building up memory.